ledger/store/trackerdb/testsuite/accounts_kv_test.go

```diff
@@ -232,6 +235,146 @@ func CustomTestResourcesQueryAll(t *customT) {
 	require.Equal(t, expectedRound, rnd)          // db round (from the return)
 }
 
+func CustomTestResourcesQueryAllLimited(t *customT) {
```

Comment:
Unsure where the best place for this would be, but it may be a good idea to test that the query excludes applications. E.g. an account is opted into asset `n`, app `n+1`, and asset `n+2`, and the query correctly skips the app

Reply:
Yes, needs to be added. I'm writing a migration to add CreatableType to the db table for resources - I'm not convinced its a significant enough performance hit to do repeated queries fishing as an alternative..

Reply:
(The other option being to only return non-deleted assets/go back to inner joins against the creators table - I don't think the user base would prefer that however).

Reply:
Leaving a note that it's worth testing an account opted into a deleted app as well, since this should also be filtered out

Reply:
This is exercised at the end of this exact test.

Reply:
I see this test exercising deleted assets. I'm talking about the test I suggested that makes sure apps are filtered out -- that should also ensure deleted apps are filtered out.

Reply:
Oh okay, I actually just pushed a test that includes an app being filtered out (in this case, with an associated creatable).